PyInstaller/utils/win32/winresource.py

```diff
@@ -202,7 +202,7 @@ def UpdateResources(dstpath, data, type_, names=None, languages=None):
             for language in res[type_][name]:
                 logger.info("Updating resource type %s name %s language %s",
                             type_, name, language)
-                win32api.UpdateResource(hdst, type_, name, data, language)
+                win32api.UpdateResource(hdst, type_, name, bytes(data, 'UTF-8'), language)
```

Comment:
The last thing to add is that because the XML does contain an `encoding='utf-8'` attribute, we're obliged to encode the string as UTF-8.